Apps/Sandcastle/gallery/Image-Based Lighting.html

```diff
@@ -145,6 +145,8 @@
           })
         );
 
+        const ibl = model.imageBasedLighting;
```

Comment:
If this is only used inside the callback, might as well put it just before the first use, if only for easier readability.